simulator/kruxsim/devices.py

```diff
@@ -36,48 +36,76 @@
     DOCK: (440, 820),
 }
 
-
-def with_prefix(device):
+##################################
```

Comment:
Is this docs or a simple comment? if it is docs, we should use triple quotes `"""`